lib/AST/ASTVerifier.cpp

```diff
@@ -492,6 +492,7 @@ class Verifier : public ASTWalker {
       case AbstractFunctionDecl::BodyKind::MemberwiseInitializer:
         return true;
 
+      case AbstractFunctionDecl::BodyKind::Deserialized:
```

Comment:
This should probably be in the top bucket, since "Deserialized" is closest to "None" rather than "Unparsed".